core/src/main/java/org/hibernate/ogm/util/impl/ArrayHelper.java

```diff
@@ -86,4 +87,20 @@ public static boolean contains(Object[] array, Object element) {
 
 		return result;
 	}
+
+	public static <T> T[] concat(List<T[]> arrays) {
```

Comment:
For some queries I need to pass as parameters an array of columns the are not always in one key, Somtimes the values are in the AssociationKey, EntityKey and a third array. This method join these arrays in a unique one that I can pass as parameter. I will add the javadoc.

Not sure if there is a way to avoid it.
I will also open a JIRA to add unit tests for this class since they are missing for all the methods.